src/Swarm/Language/Syntax.hs

```diff
@@ -539,6 +541,10 @@ constInfo c = case c of
       [ "Leaves behind a growing seed if the harvested item is growable."
       , "Otherwise it works exactly like `grab`."
       ]
+  Ignite ->
+    command 1 short . doc "Ignite a combustible item in the specified direction." $
+      [ "Combustion persists for a random duration and may spread."
```

Comment:
Can we say something about what the return value represents?

Reply:
Ah, it is not supposed to have a return value.  Good catch.